tidb-cloud/migrate-from-aurora-bulk-import.md

```diff
@@ -120,7 +119,6 @@ You need to prepare an EC2 to run the following data export task. It's better to
     --threads 8 \
     -o "$backup_dir" \
     --consistency="none" \
-    --s3.region="$s3_bucket_region" \
```

Comment:
@WangLe1321 Do we need to remove the s3.region parameter from TiDB cloud docs?

Reply:
I think we can retain `s3.region` here. User can install any version of dumpling by tiup. If the version of dumpling is lower than v6.2.0, it also need `s3.region` parameter.